src/app/pages/update-profile-request/components/profile-box.tsx

```diff
@@ -0,0 +1,73 @@
+import { Profile } from '@stacks/profile';
+import { color, Stack } from '@stacks/ui';
+
+function Value({ v }: { v: any }) {
+  if (typeof v === 'string' || typeof v === 'number' || typeof v === 'boolean') {
+    return <span style={{ wordBreak: 'break-all' }}>{v}</span>;
+  } else if (v instanceof Array) {
+    return (
+      <>
+        [
+        <div style={{ textIndent: '0em' }}>
+          {v.map((entry, index) => {
+            return (
+              <div key={index}>
+                <Value v={entry} />
+                {index < v.length - 1 && <>,</>}
+              </div>
+            );
+          })}
+        </div>
+        ]
+      </>
+    );
+  } else if (typeof v === 'object') {
+    return <Properties p={v} />;
+  } else {
+    return <>{JSON.stringify(v)}</>;
+  }
+}
+
+function Properties({ p }: { p: any }) {
+  return (
+    <>
+      {Object.keys(p)
+        .filter(k => !k.startsWith('@'))
+        .map(k => {
+          return (
+            <div key={k} style={{ textIndent: '-1em', marginLeft: '1em' }}>
+              <span style={{ color: color('text-caption') }}>{k}:</span> <Value v={p[k]} />
+            </div>
+          );
+        })}
+    </>
+  );
+}
+
+export function ProfileBox({ profile }: { profile: Profile }): JSX.Element | null {
+  return (
+    <>
+      <Stack minHeight={'260px'}>
```

Comment:
A little confused why there are so many `Stack` elements used here?

Reply:
I copied from structured message signature flow.
I updated it now to use `Box` instead.